src/binding.cc

```diff
@@ -1484,6 +1484,11 @@ const v8::Object* v8__Function__NewInstance(const v8::Function& self,
       ptr_to_local(&context), argc, const_ptr_array_to_local_array(argv)));
 }
 
+const v8::Signature* v8__Signature__New(v8::Isolate* isolate,
+                                        const v8::FunctionTemplate& templ) {
+  return local_to_ptr(v8::Signature::New(isolate, ptr_to_local(&templ)));
```

Comment:
```suggestion
                                        const v8::FunctionTemplate* templ) {
  return local_to_ptr(v8::Signature::New(isolate, ptr_to_local(templ)));
```
Done, but I'll note that rusty_v8 is pretty schizophrenic when it comes to pointers-vs-references. Maybe I'll open a PR to switch everything over to pointers.